pages/team.html

```diff
@@ -0,0 +1,147 @@
+<script>
+	{
+		"title": "The jQuery Team",
+		"customFields": [
+			{
+				"key": "subtitle",
+				"value": "The jQuery Project is run by a distributed group of volunteers."
+			},{
+				"key": "full-width",
+				"value": 1
+			}
+		]
+	}
+</script>
+
+<h1 class="block">The jQuery Team</h1>
+
+<ul class="block-grid five-up mobile-two-up team-list">
+	<li>
+		<img src="https://secure.gravatar.com/avatar/f16d4602c1c90646438a0b534ff61889?size=120" alt="">
+		<h2>Richard Gibson</h2>
+		<h3>Sizzle Lead | Core | QUnit</h3>
+	</li>
+	<li>
+		<img src="https://secure.gravatar.com/avatar/72ea0622f7b690ed82f5261b16ff8cef?size=120" alt="">
+		<h2>Michał Gołębiowski-Owczarek</h2>
+		<h3>Core</h3>
```

Comment:
OK, I guess since we list Felix then let's add `| UI` to myself as well; otherwise, it will look like Felix is the only one involved in UI in any way at the moment.

Reply:
Done.